### PR TITLE
Storybook: Add vision statement to intro page

### LINKED ISSUE #27198: Storybook: Add vision statement to intro page
Add the vision text that the design system knights came up with to the storybook intro page.
The vision text should clarify why we have a design system, what we are hoping to get out of it and how people can get involved.

### PR DESCRIPTION
**Pull request by jessover9000**
Added the vision statement compiled by the Grafana Labs design system knights to the grafana-ui intro section. The vision statement gives a clearer idea why grafana-ui exists and how it can serve our users. Feedback and further improvement ideas welcome!

### CODE REVIEW

File: packages/grafana-ui/src/Intro.story.mdx [hunk 1]
```diff
@@ -1,12 +1,26 @@
...
-@grafana/ui is a collection of components used by [Grafana](https://github.com/grafana/grafana)
+With the design system @grafana/ui, we want to democratize development. This library of reusable [Grafana](https://github.com/grafana/grafana) components and guidelines helps you with contribution and plugin development.
 
-If you're a plugin developer, you can use these component to give your plugin an official look-and-feel.
+## A library of unified components and styles
+Grafana-ui is a library of reusable components and guidelines on how to use and develop them. You can arrange them into any complex interface you need, for example in your Grafana plugin, streamlining your design process.
+
+**Our vision**
+Grafana Labs started grafana-ui to make contributing to Grafana as easy as possible for Grafanistas and community members of all fields. We want to create a component library that results in
+1. Understanding of how each component works and how you can use it to create a great user experience.
```
Comment (oddlittlebird) [inline packages/grafana-ui/src/Intro.story.mdx:14]: This should be an unordered (bulleted) list.

File: packages/grafana-ui/src/Intro.story.mdx [hunk 2]
```diff
@@ -1,12 +1,26 @@
...
 > **@grafana/ui is currently in BETA**.
 
-@grafana/ui is a collection of components used by [Grafana](https://github.com/grafana/grafana)
+With the design system @grafana/ui, we want to democratize development. This library of reusable [Grafana](https://github.com/grafana/grafana) components and guidelines helps you with contribution and plugin development.
 
-If you're a plugin developer, you can use these component to give your plugin an official look-and-feel.
+## A library of unified components and styles
+Grafana-ui is a library of reusable components and guidelines on how to use and develop them. You can arrange them into any complex interface you need, for example in your Grafana plugin, streamlining your design process.
+
+**Our vision**
```
Comment (oddlittlebird) [inline packages/grafana-ui/src/Intro.story.mdx:12]: Please turn the bold subheadings in this section into h3s.

**Review by oddlittlebird — APPROVED** [review]
Great stuff. Just a couple minor format things. Please coordinate with @marcusolsson regarding crosslinking from plugin developer docs.

**Review by kaydelaney — APPROVED** [review]
LGTM!

File: packages/grafana-ui/src/Intro.story.mdx [hunk 3]
```diff
@@ -1,12 +1,26 @@
...
+# Grafana design system
 
 > **@grafana/ui is currently in BETA**.
 
-@grafana/ui is a collection of components used by [Grafana](https://github.com/grafana/grafana)
+With the design system @grafana/ui, we want to democratize development. This library of reusable [Grafana](https://github.com/grafana/grafana) components and guidelines helps you with contribution and plugin development.
 
-If you're a plugin developer, you can use these component to give your plugin an official look-and-feel.
+## A library of unified components and styles
+Grafana-ui is a library of reusable components and guidelines on how to use and develop them. You can arrange them into any complex interface you need, for example in your Grafana plugin, streamlining your design process.
```
Comment (Clarity-89) [inline packages/grafana-ui/src/Intro.story.mdx:10]: I think this paragraph basically repeats the previous one and maybe could be removed.

File: packages/grafana-ui/src/Intro.story.mdx [hunk 4]
```diff
@@ -1,12 +1,26 @@
...
+
+**Our vision**
+Grafana Labs started grafana-ui to make contributing to Grafana as easy as possible for Grafanistas and community members of all fields. We want to create a component library that results in
```
Comment (Clarity-89) [inline packages/grafana-ui/src/Intro.story.mdx:13]:
```suggestion
Grafana Labs started grafana-ui to make contributing to Grafana as easy as possible for Grafanistas and community members of all fields. We want to create a component library that results in:
```

File: packages/grafana-ui/src/Intro.story.mdx [hunk 5]
```diff
@@ -1,12 +1,26 @@
...
+1. Understanding of how each component works and how you can use it to create a great user experience. 
+2. Short development times and consistent code quality.
+3. A beautiful, visually consistent Grafana experience.
+4. Transparency about how we work and what we do.
+
+**Maintained by Grafana Labs and you**
+Grafana Labs has a task force that helps create and maintain components. We make sure that components are documented and easy to use. The current status of the grafana-ui development is available on [GitHub](https://github.com/grafana/grafana/projects/26). Feel free to contribute!
```
Comment (Clarity-89) [inline packages/grafana-ui/src/Intro.story.mdx:20]: Should it be `@grafana-ui` throughout the text, the same as it is in the first paragraph?

File: packages/grafana-ui/src/Intro.story.mdx [hunk 6]
```diff
@@ -1,12 +1,26 @@
...
+
+**How to get involved**
+When we notice that we need to change something, we ideate on what the change should be, then we put the change in place and communicate it publicly. Developers and designers create and improve grafana-ui together. Throughout the process, we strive to involve you and meet your needs. We are looking forward to discussing your design and improvement ideas on [GitHub](https://github.com/grafana/grafana/projects/26).
```
Comment (Clarity-89) [inline packages/grafana-ui/src/Intro.story.mdx:23]: Not sure if it's just me but `ideate` sounds a bit weird. I'd use smth like `brainstorm`.

**Pull request merged**